js/gatecoin.js

```diff
@@ -264,6 +265,14 @@ module.exports = class gatecoin extends Exchange {
         return this.parseOrderBook (orderbook, undefined, 'bids', 'asks', 'price', 'volume');
     }
 
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.exchange.privateGetTradeOrdersOrderID (this.exchange.extend ({
```

Comment:
`this.exchange.` → `this.` ) also, a few lines below )

Reply:
sorry, before creating merge request I create a test in our project that uses this.exchange to invoke original functions. I have just fixed.

Reply:
No worries, I'll merge it anyway, just listing out the obvious issues with it to help clean it up...